CyLR/src/Arguments.cs

```diff
@@ -41,6 +41,10 @@ public class Arguments
             {
                 "--force-native",
                 "Uses the native file system instead of a raw NTFS read. Unix-like environments always use this option."
+            },
+            {
+                "--zp",
+                "Uses a password to encrypt the archive file"
```

Comment:
This is inconsistent with the case statement (two dashes instead of one). I'd say -zp and/or --zip-password should be the flag.